dgraph/sample/data.json

```diff
@@ -14,23 +14,219 @@
           "value": "6.2.1 Beta-lactam medicines"
         },
         {
-          "type": "unspc_code",
-          "value": "51281712"
+          "type": "drugbank_code",
+          "value": "DB01060",
+          "has_property": [
+            {
+              "type": "source",
+              "value": "https://drugbank.ca/drugs/" 
+            }
+          ]
         },
         {
-          "type": "usfda_code",
-          "value": "544Y3K6MYH"
+          "type": "atc_code",
+          "value": "J01CA04"
         },
         {
-          "type": "drugbank_code",
-          "value": "DB01060"
+          "type": "synonym",
+          "value": "Amoxicillin Anhydrous"
         },
         {
-          "type": "atc_code",
-          "value": "J01CA04"
+          "type": "synonym",
+          "value": "Amoxicillin trihydrate",
+          "has_property": [
+            {
+              "type": "unspc_code",
+              "value": "51281712"
+            }
+          ]
+        },
+        {
+          "type": "synonym",
+          "value": "Amoxicillin sodium",
+          "has_property": [
+            {
+              "type": "usfda_code",
+              "value": "544Y3K6MYH"
+            }
+          ]
         }
       ],
-      "has_child": []
+      "has_child": [
+        {
+          "description": "Amoxicillin oral",
+          "type": "form_category",
```

Comment:
Wondering if we should add an arbitrary code here? Think would be tidier if all entity nodes had a `code`?

Reply:
Yeah makes sense - I guess we're still a bit unsure exactly which will and won't have codes, but we'd definitely want to make sure we can handle them if they exist.

Added in 622f2d4724afccb5ae04556103a8e5b003b21c90